### PR TITLE
chore(option): add value example

### PR DESCRIPTION
**Pull request by rinaok**

### CODE REVIEW

**Comment by olaf-k** [comment]
ok, documenting `value` is a good initiative indeed, but for the sake of consistency I'd have 2 comments:
1. only add one example with `value`, as we do in other pages (1 attribute <-> 1 example), not in all of them.
2. extend the PR to make sure all form elements are covered (I'm thinking `checkbox` and `radio`; `switch` already has an entry, maybe I'm missing others?)

**Comment by rinaok** [comment]
> ok, documenting `value` is a good initiative indeed, but for the sake of consistency I'd have 2 comments:
> 
> 1. only add one example with `value`, as we do in other pages (1 attribute <-> 1 example), not in all of them.
> 2. extend the PR to make sure all form elements are covered (I'm thinking `checkbox` and `radio`; `switch` already has an entry, maybe I'm missing others?)

@olaf-k done

**Comment by olaf-k** [comment]
> @olaf-k done

you missed the `slider`, it was a test 
SasEvilGIF

**Review by olaf-k — COMMENTED** [review]
👍

**Review by olaf-k — APPROVED** [review]

File: libs/components/src/lib/slider/README.md [hunk 1]
```diff
@@ -100,3 +100,14 @@ Toggle the `disabled` member to disable/enable the slider.
...
+### Value
+
+Use `value` in order to set the value of the switch.
+
+- Type: `string`
+- Default: `on`
+
+```html
+<vwc-switch value="my-value"></vwc-switch>
+```
```
Comment (olaf-k) [inline libs/components/src/lib/slider/README.md:113]:
```suggestion
Use `value` in order to set the value of the slider.

- Type: `string`
- Default: `on`

```html
<vwc-slider value="5"></vwc-slider>
```
```

**Review by olaf-k — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/slider/README.md [hunk 2]
```diff
@@ -100,3 +100,13 @@ Toggle the `disabled` member to disable/enable the slider.
 ```html preview blocks
 <vwc-slider disabled></vwc-slider>
 ```
+
+### Value
+
+Use `value` in order to set the value of the slider.
+
+- Type: `string`
+- Default: `on`
```
Comment (olaf-k) [inline libs/components/src/lib/slider/README.md:109]:
```suggestion
- Default: `5`
```
(missed that earlier)

**Review by olaf-k — CHANGES_REQUESTED** [review]

**Review by olaf-k — APPROVED** [review]

**Pull request merged**